mmv1/third_party/terraform/website/docs/d/composer_user_workloads_config_map.html.markdown

```diff
@@ -0,0 +1,56 @@
+---
+subcategory: "Cloud Composer"
+description: |-
+  Kubernetes ConfigMap in Composer Environment workloads.
```

Comment:
I would make it consistent with changes that we did for User Workloads Secrets documentation, using wording from Composer API docs
https://github.com/GoogleCloudPlatform/magic-modules/pull/10490/files

Reply:
fixed